docs/REST API/rest-api.md

```diff
@@ -3278,67 +3279,124 @@ Retrieves the aggregated host OS versions information.
 | platform            | string   | query | Filters the hosts to the specified platform |
 | os_name     | string | query | The name of the operating system to filter hosts by. `os_version` must also be specified with `os_name`                                                 |
 | os_version    | string | query | The version of the operating system to filter hosts by. `os_name` must also be specified with `os_version`                                                 |
+| page                    | integer | query | Page number of the results to fetch.                                                                                                                                       |
+| per_page                | integer | query | Results per page.                                                                                                                                                          |
+| order_key               | string  | query | What to order results by. Allowed fields are: `hosts_count`. Default is `hosts_count` (descending).      |
+| order_direction | string | query | **Requires `order_key`**. The direction of the order given the order key. Options include `asc` and `desc`. Default is `asc`. |
+
 
 ##### Default response
 
 `Status: 200`
 
 ```json
 {
-  "counts_updated_at": "2022-03-22T21:38:31Z",
+  "counts_updated_at": "2023-12-06T22:17:30Z",
```

Comment:
@ghernandez345 does this look correct?  @rachaelshaw a total count is needed here for the frontend.

```suggestion
  "count": 1
  "counts_updated_at": "2023-12-06T22:17:30Z",
```